Repo.rb

```diff
@@ -144,8 +144,18 @@ def system_call_git(*args)
     system("git #{args_string}")
   end
 
-  def diff(*args)
-    result = with_captured_stdout { @gitrepo.diff }
+  def branch(git_str)
+    output = git_command_here('branch', git_str)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.